documentation/specs/src/interoperability/ethereum-bridge/ethereum_events_attestation.md

```diff
@@ -35,40 +49,71 @@ on Namada. This minimum number of confirmations will be changeable via
 governance.
 
 `TransferToNamada` events may include a custom minimum number of
-confirmations, that must be at least the protocol-specified minimum number of
-confirmations but is initially set to __100__.
+confirmations that must be at least the protocol-specified minimum number of
+confirmations. However, this value is initially set to __100__.
+
+Validators must not vote to include events that have not met the required
+number of confirmations. Votes on unconfirmed events will eventually time
+out in storage, unless the number of confirmations was only off by a few
+block heights in Ethereum. Assuming that an honest majority of validators
```

Comment:
What's the honest majority threshold here? Should be <1/3 Byzantine, as usual, right?

Reply:
yep

Reply:
We should clarify that, then